### ISSUE TITLE
Amend schema to accomodate curation-based meta-platforms

### ISSUE THREAD

**Issue by neatonk** [neume-network/schema#34]
With curation-based platforms—like present materials and release club—comes the possibility of any NFT being associated with multiple platforms. The schema does not currently support this, as each NFT is associated with a single platform. There are multiple ways of thinking about this and some discussion will be needed to find the right solution.

For starters:
- What is a platform actually? And what do we gain by grouping music NFTs in this way?
- Is a curation-based platform actually a platform based on the answer to the previous question? And is it useful in the same way?
- What about nested curation-based platforms representing multiple levels of curation and so on? Is this still in scope for neume?
- Given our answers to all of the above, how might we represent curation-based platforms in this schema? And what use cases does this serve?

**Comment by TimDaub** [comment]
> * Given our answers to all of the above, how might we represent curation-based platforms in this schema? And what use cases does this serve?

I think this strongly depends on the signals of the curation platform. E.g. if we had a platform that was aggregating likes e.g. musicos, then I think it'd be interesting if the neume network track object would feature the likes as a stat somewhere.

**Comment by neatonk** [comment]
> I think this strongly depends on the signals of the curation platform. E.g. if we had a platform that was aggregating likes e.g. musicos, then I think it'd be interesting if the neume network track object would feature the likes as a stat somewhere.

Right, so the semantics of the curation platform matter. E.g. does this curation signal represent a like, a release, or something else.

**Comment by neatonk** [comment]
My goal in this issue was to find a way to permit multiple platforms per NFT. E.g. in Future Tape, present materials is represented as a tag. Since each track can have many tags this easily supports many platforms per track via tags. This does not affect the minting platform, of which there can be only one.

I like the flexibility of this approach and think it could work well for neume and musicOS too. That said, something a bit richer than string tags may be useful. E.g. to identify that a tag represents a platform.
